package.json

```diff
@@ -12,15 +12,17 @@
     "codegen:graphql": "graphql-codegen",
     "codegen:webhooks": "sh ./scripts/codegen-webhooks.sh",
     "typecheck": "tsc --noEmit",
-    "lint": "pnpm run lint:eslint && pnpm run lint:prettier",
-    "lint:eslint": "eslint 'src/**/*.ts'",
-    "lint:prettier": "prettier --config=.prettierrc --ignore-path=.prettierignore --check '**/*.{ts,tsx,gql,js}'",
+    "lint": "biome lint .",
+    "lint:fix": "biome lint . --fix",
+    "format": "biome check .",
```

Comment:
check runs lint, as far as I know this means lint is superfluous - you can get rid of it.